### DISCUSSION TITLE
Simultaneous $wire method calls always return the value of the last call

### DISCUSSION THREAD

**Discussion by danharrin** [livewire/livewire#3065] [7. Bugs (archived)] [answered]
### Description

Calling the same method simultaneously always returns the value of the last call.

This is easily testable by passing in different parameters to each call and watch them be ignored by Livewire.

### Exact steps to reproduce

https://laravelplayground.com/#/snippets/72425c04-9a29-4272-9c7b-ca5eb74c4604

**Context**
- Livewire version: 2.4.0
- Laravel version: 8.5.12
- Alpine version: 2.8.1
- Browser: Safari Technology Preview

**Comment by danharrin** [comment]
Can confirm that this is still in issue in the latest version of Alpine and Livewire.

**Comment by joshhanley** [comment, marked as answer]
For anyone that finds this, this has been fixed in #3510
